### DISCUSSION TITLE
Welcome to Vue2Editor Discussions!

### DISCUSSION THREAD

**Discussion by davidroyer** [davidroyer/vue2-editor#335] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us what you'd like to see in our Discussions Community.

**Comment by InfoLibre** [comment]
Hello, I'm using https://github.com/surmon-china/vue-quill-editor in https://framagit.org/InfoLibre/rapido
But I'm not sure it was a good choice because it seems not to be updated.
I'm looking at Vue2Editor. Would it be a better choise ?

Did you try https://github.com/xdan/jodit or https://github.com/WendellAdriel/jodit-vue ? Would it be a better editor than Quill ?
